# FILES ARE NOT PULLED FROM THE BUCKET: "was unable to get file... aws not found". The aws not found command is shell's equivalent to bash's "command not found", but tried manually the image and aws-cli is indeed installed

+ POD_NAME=train-mpijob-worker-0
+ [ t = - ]
+ shift
+ /opt/kube/kubectl cp /opt/kube/hosts train-mpijob-worker-0:/etc/hosts_of_nodes
+ POD_NAME=train-mpijob-worker-1
+ [ t = - ]
+ shift
+ /opt/kube/kubectl cp /opt/kube/hosts train-mpijob-worker-1:/etc/hosts_of_nodes
+ /opt/kube/kubectl exec train-mpijob-worker-0 -- /bin/sh -c cat /etc/hosts_of_nodes >> /etc/hosts &&     PATH=/usr/local/bin:$PATH ; export PATH ; LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH ; export LD_LIBRARY_PATH ; DYLD_LIBRARY_PATH=/usr/local/lib:$DYLD_LIBRARY_PATH ; export DYLD_LIBRARY_PATH ;   /usr/local/bin/orted -mca ess "env" -mca ess_base_jobid "3580297216" -mca ess_base_vpid 1 -mca ess_base_num_procs "3" -mca orte_node_regex "kubenode-opentelekomcloud-dltest-xtmf-0,train-mpijob-worker-0,train-mpijob-worker-1@0(3)" -mca orte_hnp_uri "3580297216.0;tcp://192.168.0.7,172.17.0.1,10.244.0.0,10.244.0.1:39747" -mca pml "ob1" -mca btl "^openib" -mca plm "rsh" -mca plm_rsh_agent "/etc/mpi/kubexec.sh" -mca orte_default_hostfile "/etc/mpi/hostfile" -mca orte_tag_output "1" -mca pmix "^s1,s2,cray,isolated"
+ /opt/kube/kubectl exec train-mpijob-worker-1 -- /bin/sh -c cat /etc/hosts_of_nodes >> /etc/hosts &&     PATH=/usr/local/bin:$PATH ; export PATH ; LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH ; export LD_LIBRARY_PATH ; DYLD_LIBRARY_PATH=/usr/local/lib:$DYLD_LIBRARY_PATH ; export DYLD_LIBRARY_PATH ;   /usr/local/bin/orted -mca ess "env" -mca ess_base_jobid "3580297216" -mca ess_base_vpid 2 -mca ess_base_num_procs "3" -mca orte_node_regex "kubenode-opentelekomcloud-dltest-xtmf-0,train-mpijob-worker-0,train-mpijob-worker-1@0(3)" -mca orte_hnp_uri "3580297216.0;tcp://192.168.0.7,172.17.0.1,10.244.0.0,10.244.0.1:39747" -mca pml "ob1" -mca btl "^openib" -mca plm "rsh" -mca plm_rsh_agent "/etc/mpi/kubexec.sh" -mca orte_default_hostfile "/etc/mpi/hostfile" -mca orte_tag_output "1" -mca pmix "^s1,s2,cray,isolated"
[1,1]<stdout>:on train-mpijob-worker-1 with sub-rank 0
[1,0]<stdout>:on train-mpijob-worker-0 with sub-rank 0
[1,0]<stdout>:Sharing a node and taking on the gpu
[1,0]<stdout>:rank 0 can have gpu0
[1,1]<stdout>:Sharing a node and taking on the gpu
[1,1]<stdout>:rank 1 can have gpu0
[1,0]<stdout>:set to device 0 train-mpijob-worker-0
[1,0]<stdout>:tensorflow
[1,1]<stdout>:set to device 0 train-mpijob-worker-1
[1,1]<stdout>:tensorflow
[1,1]<stderr>:2020-09-10 16:25:47.431829: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
[1,0]<stderr>:2020-09-10 16:25:47.438798: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
[1,1]<stderr>:2020-09-10 16:25:47.656772: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:897] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
[1,1]<stderr>:2020-09-10 16:25:47.657857: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1405] Found device 0 with properties: 
[1,1]<stderr>:name: Tesla V100-PCIE-16GB major: 7 minor: 0 memoryClockRate(GHz): 1.38
[1,1]<stderr>:pciBusID: 0000:00:0d.0
[1,1]<stderr>:totalMemory: 15.78GiB freeMemory: 15.48GiB
[1,1]<stderr>:2020-09-10 16:25:47.657901: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1484] Adding visible gpu devices: 0
[1,0]<stderr>:2020-09-10 16:25:47.675856: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:897] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
[1,0]<stderr>:2020-09-10 16:25:47.676933: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1405] Found device 0 with properties: 
[1,0]<stderr>:name: Tesla V100-PCIE-16GB major: 7 minor: 0 memoryClockRate(GHz): 1.38
[1,0]<stderr>:pciBusID: 0000:00:0d.0
[1,0]<stderr>:totalMemory: 15.78GiB freeMemory: 15.48GiB
[1,0]<stderr>:2020-09-10 16:25:47.676980: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1484] Adding visible gpu devices: 0
[1,1]<stderr>:2020-09-10 16:25:47.982500: I tensorflow/core/common_runtime/gpu/gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
[1,1]<stderr>:2020-09-10 16:25:47.982574: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971]      0 
[1,1]<stderr>:2020-09-10 16:25:47.982585: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] 0:   N 
[1,1]<stderr>:2020-09-10 16:25:47.982747: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 14977 MB memory) -> physical GPU (device: 0, name: Tesla V100-PCIE-16GB, pci bus id: 0000:00:0d.0, compute capability: 7.0)
[1,1]<stdout>:Process 1 using device /gpu:0
[1,1]<stdout>:copying EleEscan_EleEscan_1_1.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_1.h5.h5
[1,0]<stderr>:2020-09-10 16:25:48.014620: I tensorflow/core/common_runtime/gpu/gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
[1,0]<stderr>:2020-09-10 16:25:48.014688: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971]      0 
[1,0]<stderr>:2020-09-10 16:25:48.014703: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] 0:   N 
[1,0]<stderr>:2020-09-10 16:25:48.014869: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 14977 MB memory) -> physical GPU (device: 0, name: Tesla V100-PCIE-16GB, pci bus id: 0000:00:0d.0, compute capability: 7.0)
[1,0]<stdout>:Process 0 using device /gpu:0
[1,1]<stderr>:sh: 1: aws: not found
[1,1]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_1_1.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_1.h5.h5
[1,1]<stdout>:copying EleEscan_EleEscan_1_2.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_2.h5.h5
[1,1]<stderr>:sh: 1: aws: not found
[1,1]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_1_2.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_2.h5.h5
[1,1]<stdout>:copying EleEscan_EleEscan_1_3.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_3.h5.h5
[1,1]<stderr>:sh: 1: aws: not found
[1,1]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_1_3.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_3.h5.h5
[1,1]<stdout>:copying EleEscan_EleEscan_1_4.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_4.h5.h5
[1,0]<stdout>:copying EleEscan_EleEscan_1_1.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_1.h5.h5
[1,1]<stderr>:sh: 1: aws: not found
[1,1]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_1_4.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_4.h5.h5
[1,1]<stdout>:copying EleEscan_EleEscan_1_5.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_5.h5.h5
[1,0]<stderr>:sh: 1: aws: not found
[1,0]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_1_1.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_1.h5.h5
[1,0]<stdout>:copying EleEscan_EleEscan_1_2.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_2.h5.h5
[1,1]<stderr>:sh: 1: [1,1]<stderr>:aws: not found
[1,1]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_1_5.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_5.h5.h5
[1,1]<stdout>:copying EleEscan_EleEscan_1_6.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_6.h5.h5
[1,0]<stderr>:sh: 1: aws: not found
[1,0]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_1_2.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_2.h5.h5
[1,0]<stdout>:copying EleEscan_EleEscan_1_3.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_3.h5.h5
[1,1]<stderr>:sh: 1: aws: not found
[1,1]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_1_6.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_6.h5.h5
[1,1]<stdout>:copying EleEscan_EleEscan_1_7.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_7.h5.h5
[1,0]<stderr>:sh: 1: aws: not found
[1,0]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_1_3.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_3.h5.h5
[1,0]<stdout>:copying EleEscan_EleEscan_1_4.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_4.h5.h5
[1,1]<stderr>:sh: 1: aws: not found
[1,1]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_1_7.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_7.h5.h5
[1,1]<stdout>:copying EleEscan_EleEscan_1_8.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_8.h5.h5
[1,0]<stderr>:sh: 1: aws: not found
[1,0]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_1_4.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_4.h5.h5
[1,0]<stdout>:copying EleEscan_EleEscan_1_5.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_5.h5.h5
[1,1]<stderr>:sh: 1: aws: not found
[1,1]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_1_8.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_8.h5.h5
[1,1]<stdout>:copying EleEscan_EleEscan_1_9.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_9.h5.h5
[1,1]<stderr>:sh: 1: aws: not found
[1,1]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_1_9.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_9.h5.h5
[1,1]<stdout>:copying EleEscan_EleEscan_3_1.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_1.h5.h5
[1,0]<stderr>:sh: 1: aws: not found
[1,0]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_1_5.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_5.h5.h5
[1,0]<stdout>:copying EleEscan_EleEscan_1_6.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_6.h5.h5
[1,1]<stderr>:sh: 1: aws: not found
[1,1]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_3_1.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_1.h5.h5
[1,1]<stdout>:copying EleEscan_EleEscan_3_2.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_2.h5.h5
[1,0]<stderr>:sh: 1: aws: not found
[1,0]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_1_6.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_6.h5.h5
[1,0]<stdout>:copying EleEscan_EleEscan_1_7.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_7.h5.h5
[1,1]<stderr>:sh: 1: aws: not found
[1,0]<stderr>:sh: 1: [1,0]<stderr>:aws: not found[1,0]<stderr>:
[1,0]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_1_7.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_7.h5.h5
[1,0]<stdout>:copying EleEscan_EleEscan_1_8.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_8.h5.h5
[1,1]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_3_2.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_2.h5.h5
[1,1]<stdout>:copying EleEscan_EleEscan_3_3.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_3.h5.h5
[1,1]<stderr>:sh: 1: aws: not found
[1,1]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_3_3.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_3.h5.h5
[1,1]<stdout>:copying EleEscan_EleEscan_3_4.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_4.h5.h5
[1,0]<stderr>:sh: 1: aws: not found
[1,0]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_1_8.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_8.h5.h5
[1,0]<stdout>:copying EleEscan_EleEscan_1_9.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_9.h5.h5
[1,1]<stderr>:sh: 1: aws: not found
[1,1]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_3_4.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_4.h5.h5
[1,1]<stdout>:copying EleEscan_EleEscan_3_5.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_5.h5.h5
[1,0]<stderr>:sh: 1: aws: not found
[1,0]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_1_9.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_1_9.h5.h5
[1,0]<stdout>:copying EleEscan_EleEscan_3_1.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_1.h5.h5
[1,1]<stderr>:sh: 1: aws: not found
[1,1]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_3_5.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_5.h5.h5
[1,1]<stdout>:copying EleEscan_EleEscan_3_6.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_6.h5.h5
[1,0]<stderr>:sh: 1: aws: not found
[1,0]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_3_1.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_1.h5.h5
[1,0]<stdout>:copying EleEscan_EleEscan_3_2.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_2.h5.h5
[1,1]<stderr>:sh: 1: aws: not found
[1,1]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_3_6.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_6.h5.h5
[1,1]<stdout>:copying EleEscan_EleEscan_3_7.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_7.h5.h5
[1,0]<stderr>:sh: 1: aws: not found
[1,0]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_3_2.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_2.h5.h5
[1,0]<stdout>:copying EleEscan_EleEscan_3_3.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_3.h5.h5
[1,1]<stderr>:sh: 1: aws: not found
[1,1]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_3_7.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_7.h5.h5
[1,1]<stdout>:copying EleEscan_EleEscan_3_8.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_8.h5.h5
[1,0]<stderr>:sh: 1: aws: not found
[1,0]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_3_3.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_3.h5.h5
[1,0]<stdout>:copying EleEscan_EleEscan_3_4.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_4.h5.h5
[1,1]<stderr>:sh: 1: aws: not found
[1,1]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_3_8.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_8.h5.h5
[1,1]<stdout>:copying EleEscan_EleEscan_3_10.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_10.h5.h5
[1,0]<stderr>:sh: 1: aws: not found
[1,0]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_3_4.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_4.h5.h5
[1,0]<stdout>:copying EleEscan_EleEscan_3_5.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_5.h5.h5
[1,1]<stderr>:sh: 1: aws: not found
[1,1]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_3_10.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_10.h5.h5
[1,1]<stdout>:copying EleEscan_EleEscan_2_3.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_2_3.h5.h5
[1,0]<stderr>:sh: 1: aws: not found
[1,0]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_3_5.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_5.h5.h5
[1,0]<stdout>:copying EleEscan_EleEscan_3_6.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_6.h5.h5
[1,1]<stderr>:sh: 1: aws: not found
[1,1]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_2_3.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_2_3.h5.h5
[1,1]<stderr>:/usr/local/lib/python3.5/dist-packages/h5py/__init__.py:34: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
[1,1]<stderr>:  from ._conv import register_converters as _register_converters
[1,1]<stderr>:Using TensorFlow backend.
[1,0]<stderr>:sh: 1: aws: not found
[1,0]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_3_6.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_6.h5.h5
[1,0]<stdout>:copying EleEscan_EleEscan_3_7.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_7.h5.h5
[1,1]<stderr>:Traceback (most recent call last):
[1,1]<stderr>:  File "MPIGDriver.py", line 148, in <module>
[1,1]<stderr>:    validate_every = int(data.count_data()/args.batch )
[1,1]<stderr>:  File "/mpi_learn/mpi_learn/train/data.py", line 240, in count_data
[1,1]<stderr>:    h5_file = h5py.File( in_file_name, 'r' )
[1,1]<stderr>:  File "/usr/local/lib/python3.5/dist-packages/h5py/_hl/files.py", line 271, in __init__
[1,1]<stderr>:    fid = make_fid(name, mode, userblock_size, fapl, swmr=swmr)
[1,1]<stderr>:  File "/usr/local/lib/python3.5/dist-packages/h5py/_hl/files.py", line 101, in make_fid
[1,1]<stderr>:    fid = h5f.open(name, flags, fapl=fapl)
[1,1]<stderr>:  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper (/tmp/pip-huypgcah-build/h5py/_objects.c:2840)
[1,1]<stderr>:  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper (/tmp/pip-huypgcah-build/h5py/_objects.c:2798)
[1,1]<stderr>:  File "h5py/h5f.pyx", line 78, in h5py.h5f.open (/tmp/pip-huypgcah-build/h5py/h5f.c:2117)
[1,1]<stderr>:OSError: Unable to open file (Unable to open file: name = 'eleescan_eleescan_1_1.h5.h5', errno = 2, error message = 'no such file or directory', flags = 0, o_flags = 0)
[1,0]<stderr>:sh: 1: aws: not found
[1,0]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_3_7.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_7.h5.h5
[1,0]<stdout>:copying EleEscan_EleEscan_3_8.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_8.h5.h5
[1,0]<stderr>:sh: 1: aws: not found
[1,0]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_3_8.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_8.h5.h5
[1,0]<stdout>:copying EleEscan_EleEscan_3_10.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_10.h5.h5
[1,0]<stderr>:sh: 1: aws: not found
[1,0]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_3_10.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_3_10.h5.h5
[1,0]<stdout>:copying EleEscan_EleEscan_2_3.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_2_3.h5.h5
[1,0]<stderr>:sh: 1: aws: not found
[1,0]<stdout>:was enable to copy the file s3://ganbucket/ EleEscan_EleEscan_2_3.h5.h5 to /tmp/gan_data183/EleEscan_EleEscan_2_3.h5.h5
[1,0]<stderr>:/usr/local/lib/python3.5/dist-packages/h5py/__init__.py:34: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
[1,0]<stderr>:  from ._conv import register_converters as _register_converters
[1,0]<stderr>:Using TensorFlow backend.
[1,0]<stderr>:Traceback (most recent call last):
[1,0]<stderr>:  File "MPIGDriver.py", line 148, in <module>
[1,0]<stderr>:    validate_every = int(data.count_data()/args.batch )
[1,0]<stderr>:  File "/mpi_learn/mpi_learn/train/data.py", line 240, in count_data
[1,0]<stderr>:    h5_file = h5py.File( in_file_name, 'r' )
[1,0]<stderr>:  File "/usr/local/lib/python3.5/dist-packages/h5py/_hl/files.py", line 271, in __init__
[1,0]<stderr>:    fid = make_fid(name, mode, userblock_size, fapl, swmr=swmr)
[1,0]<stderr>:  File "/usr/local/lib/python3.5/dist-packages/h5py/_hl/files.py", line 101, in make_fid
[1,0]<stderr>:    fid = h5f.open(name, flags, fapl=fapl)
[1,0]<stderr>:  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper (/tmp/pip-huypgcah-build/h5py/_objects.c:2840)
[1,0]<stderr>:  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper (/tmp/pip-huypgcah-build/h5py/_objects.c:2798)
[1,0]<stderr>:  File "h5py/h5f.pyx", line 78, in h5py.h5f.open (/tmp/pip-huypgcah-build/h5py/h5f.c:2117)
[1,0]<stderr>:OSError: Unable to open file (Unable to open file: name = 'eleescan_eleescan_1_1.h5.h5', errno = 2, error message = 'no such file or directory', flags = 0, o_flags = 0)
-------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
-------------------------------------------------------
--------------------------------------------------------------------------
mpirun.real detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[54631,1],0]
  Exit code:    1
--------------------------------------------------------------------------

